### PR TITLE
Update mursv1.py driver to use mem.immutable - #10288

### PR DESCRIPTION
**Pull request by KC9HI**
# CHIRP PR Checklist

The following must be true before PRs can be merged:

* All tests must be passing.
* Commits should be squashed into logical units.
* Commits should be rebased (or simply rebase-able in the web UI) on current master. Do not put merge commits in a PR.
* Commits in a single PR should be related.
* Major new features or bug fixes should reference a [CHIRP issue](https://chirp.danplanet.com/projects/chirp/issues).
* New drivers should be accompanied by a test image in `tests/images` (except for thin aliases where the driver is sufficiently tested already).

Please also follow these guidelines:

* Keep cleanups in separate commits from functional changes.
* Please write a reasonable commit message, especially if making some change that isn't totally obvious (such as adding a new model, adding a feature, etc).
* Do not add new py2-compatibility code (No new uses of `six`, `future`, etc).
* All new drivers should set `NEEDS_COMPAT_SERIAL=False` and use `MemoryMapBytes`.
* New drivers and radio models will affect the Python3 test matrix. You should regenerate this file with `tox -emakesupported` and include it in your commit.

### CODE REVIEW

File: chirp/drivers/mursv1.py [hunk 1]
```diff
@@ -67,8 +68,7 @@
 LIST_VOICE = ["Off", "English", "Chinese"]
 LIST_WORKMODE = ["Frequency", "Channel"]
 
-MURS_FREQS = [151820000, 151880000, 151940000, 154570000, 154600000] * 3
-FM_MODE = [3, 4, 8, 9, 13, 14]
+MURS_FREQS = bandplan_na.ALL_MURS_FREQS * 3
```
Comment (kk7ds) [inline chirp/drivers/mursv1.py:71]: I saw this in the other driver, what's this `* 3` about?

Comment (KC9HI) [reply chirp/drivers/mursv1.py:71]: The radio has 15 channels. 
ALL_MURS_FREQS + ALL_MURS_FREQS +ALL_MURS_FREQS or ALL_MURS_FREQS * 3

Comment (kk7ds) [reply chirp/drivers/mursv1.py:71]: Okay, but.. you don't use this list anywhere to initialize the radio do you? If you just use it for checking whether something is a MURS frequency or not, then this:

```
if mem.freq in ALL_MURS_FREQS
```

is the same whether `ALL_MURS_FREQS` has just the unique frequencies, or three copies of the unique frequencies. It's not wrong, I just don't think it's necessary to triplicate (is that a word?) them for that purpose, so I was wondering if there was some other reason.

Comment (KC9HI) [reply chirp/drivers/mursv1.py:71]: I do use it. But now that you mention it, I might be using it wrong in one or more places. Let me review it.

**Pull request merged**